server/channels/api4/system.go

```diff
@@ -181,9 +182,9 @@ func getSystemPing(c *Context, w http.ResponseWriter, r *http.Request) {
 			s[model.STATUS] = model.StatusUnhealthy
 		}
 
-		w.Header().Set(model.STATUS, s[model.STATUS])
-		w.Header().Set(dbStatusKey, s[dbStatusKey])
-		w.Header().Set(filestoreStatusKey, s[filestoreStatusKey])
+		w.Header().Set(model.STATUS, s[model.STATUS].(string))
```

Comment:
I know these didn't change, but can we add some type checks here? This will panic at once if something slides in unexpectedly.